blkn/src/Button/__tests__/index.test.js

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import Button from "../";
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
Just for my information what is this `Adapter`  for? I never used it before thanks.

Reply:
Well it turned out it was configured already in separated file  `jest.json` and it is not necessary. Removed it.